parser_library/src/processing/processing_manager.cpp

```diff
@@ -274,13 +274,12 @@ void processing_manager::finish_opencode()
 
         if (stmt->m_resemblance == context::id_storage::well_known::COPY && stmt->m_details.operands.items.size() == 1)
         {
-            if (const auto& id = hlasm_ctx_.ids().find(stmt->m_details.operands.items.front().name); id)
-                asm_processor::parse_copy(ctx_,
-                    lib_provider_,
-                    *id,
-                    stmt->m_details.operands.items.front().r,
-                    stmt->m_details.stmt_r,
-                    nullptr);
+            asm_processor::parse_copy(ctx_,
```

Comment:
Still unsure about this... the `ctx.hlasm_ctx->whole_copy_stack()` will be wrong for this call, won't it? Pointing somewhere beyond the opencode?

Reply:
It is, indeed, pointing nowhere -> `source_stack_` is actually empty. I'll try to introduce a new way how to expose the used copy members.

Reply:
This is addressed as part of #143.

Nevertheless, merge from dev is intentionally not done at this point to make this PR easier to check. Sync will be done when fuzzer is prepared for CICS.